DeskAPIClient/Podfile.lock

```diff
@@ -32,4 +32,6 @@ SPEC CHECKSUMS:
   OCMock: 1f0871af0f397183f33e7552cca088bd2b485c25
   OHHTTPStubs: cb1aefbbeb4de4e741644455d4b945538e5248a2
 
-COCOAPODS: 0.39.0
+PODFILE CHECKSUM: 6f12d3f6f6c7052330ec41b8c00d2c86d5b757f5
+
+COCOAPODS: 1.0.1
```

Comment:
We should probably update the Gemfile to reflect that this is now a Cocoapods 1.0.1 project.